packages/payment-processor/src/payment/near-conversion.ts

```diff
@@ -12,12 +12,13 @@ import {
 import { INearTransactionCallback, processNearPaymentWithConversion } from './utils-near';
 import { IConversionPaymentSettings } from '.';
 import { CurrencyManager, NearChains, UnsupportedCurrencyError } from '@requestnetwork/currency';
+import { validatePaymentReference } from '../utils/validation';
 
 /**
  * Processes the transaction to pay a request in NEAR with on-chain conversion.
  * @param request the request to pay
- * @param walletConnection the Web3 provider, or signer. Defaults to window.ethereum.
- * @param amount optionally, the amount to pay. Defaults to remaining amount of the request.
+ * @param walletConnection the Near provider.
+ * @param amount optionally, the amount to pay in request currency. Defaults to remaining amount of the request.
```

Comment:
This seems to be a correct observation - the typing in the test could use `NearProvider` instead of `any` :+1:

Reply:
I don't understand how a more precise type for a mocked value in the tests positively impacts the library. Can you explain? I'll merge this one because of emergencies and the feedback is not related to the core of my change, but happy to later improve.